src/app-nav-bar/user-menu.js

```diff
@@ -120,6 +120,7 @@ export default function UserMenuComponent(props: {|
           {...userMenuProps}
         />
       )}
+      autoFocus={false}
```

Comment:
There are multiple instances throughout the repo where we've used `Popover` in ways that assume autoFocus is false, when really it is true by default but just wasn't working correctly. In these cases we need to explicitly pass in `autoFocus={false}` to preserve existing functionality.